### DISCUSSION TITLE
Indent/Outdent of items without affect on children

### DISCUSSION THREAD

**Discussion by thuringia** [vslinko/obsidian-outliner#416] [Ideas]
**Describe the bug**
TLDR: Indent/outdent should accept a modifier like "option", to use the vanilla logic.

When refactoring outlines from meeting notes, I often encounter the problem, that splitting existing items in the outline into multiple separate items, e.g. to build a to-do list, is rather error prone because unintended items are moved.
This is a case where the vanilla indent/outdent behaviour is more desirable to me, as it allows for easy insertion of items into the hierarchy without affecting existing items.

Right now, this requires deactivation of the outliner plugin, which is cumbersome to do.

Basically, I'm looking for a way to move individual items in the outline, regardless of the parent/child relationship.
It is completely possible that this is user error, not an enhancement, unfortunately I was unable to find a way in the docs how to make this work.

**To Reproduce**

1. Create an outline item like this:
```
- foo, bar
  - baz 
```
2. Press enter after "foo" to create a new line:
```
- foo
- bar
  - baz 
```
3. Press tab to indent "foo":
```
- foo
  - bar
    - baz 
```
This indented "baz" as well, although it is a child of "foo", not "bar".

**Expected behavior**
The outline should look like this:
```
- foo
  - bar
  - baz 
```

This layout can be achieved by cycling outline items, effectively rebuilding the outline

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: macOS Ventura 13.1
 - Obsidian Version: 1.1.12

**Additional context**
n/a
